app/actions/remote/entry/common.ts

```diff
@@ -187,6 +187,7 @@ export const deferredAppEntryActions = async (
         await fetchTeamsChannelsAndUnreadPosts(serverUrl, since, teamData.teams, teamData.memberships, initialTeamId);
     }
 
+    fetchAllTeams(serverUrl);
```

Comment:
so that is one of the things I fixed here, we are only deleting the teams we are no longer a member of but WE WERE a member. So now when this is called it will add the other teams if there are any.

Reply:
But the result of this will get to the database before or after the Team Bar is shown? If it gets after, the `+` button may disappear/appear. The example will be this:
- A user with no permission to create teams.
- A instance with two public (joinable) teams.
- The user is member of both teams.
- The user close the app.
- The user exits from one of the teams from a different client.
- The user starts the app.
  - Since it is no longer a member, the team does not appear in the team sidebar
  - Since the state does not find any more joinable teams, it thinks it does not need the + button.
  - The result of fetchTeams gets into the database, and since now there is one joinable public team, the + button reappears.